posthog/api/feature_flag.py

```diff
@@ -68,6 +69,29 @@ def validate_key(self, value):
 
         return value
 
+    def validate_filters(self, filters):
+        aggregation_group_type_index = filters.get("aggregation_group_type_index", None)
```

Comment:
non-blocking

It mentions that filters isn't required [here](https://github.com/PostHog/posthog/pull/7363/files#diff-901d485ec1a60127f907399dbbf514d89d3a99456190adfa242399d61749d3c5R23), do we need to check that this isn't `None`?

non-blocking as seems like a silly case anyway

Reply:
While the link doesn't work I assume you mean `def filters` in the model.

So this is verifying the data received from FE - which will be consistent. The schema used to be different a long time ago in a way we couldn't easily fix in migrations (now probably could) so the model has a safe-guard which isn't relevant here.